### PR TITLE
[wip] Send email to new contributors

### PR DESCRIPTION
**Pull request by edenlightning**
Add new github action to send an email to all new contributors.

### CODE REVIEW

**Comment by Borda** [comment]
@edenlightning have we set the secrets already? `MAIL_USERNAME` and `MAIL_PASSWORD`

**Comment by edenlightning** [comment]
> @edenlightning have we set the secrets already? `MAIL_USERNAME` and `MAIL_PASSWORD`

no, I don't know how to do that this was in original script.

File: .github/workflows/news-contributions.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: Auto message for new contributors
+description: Automatically send a message to the first PR for new a contributor.
+branding:
+  icon: message-square
+  color: purple
```
Comment (justusschock) [inline .github/workflows/news-contributions.yml:5]: Will they still be used or can/shall we remove them?

**Comment by Borda** [comment]
lest try to update stale that planed issues/PRs won't be staled #4769

**Comment by edenlightning** [comment]
@Borda mind helping me with this? still cant figure out how GH work

**Comment by Borda** [comment]
> @Borda mind helping me with this? still cant figure out how GH work

yes, about next week.. :]

**Comment by Borda** [comment]
```
env: 
        GH_EMAIL: ${{ github.event.commits[0].author.email }}
        GH_USERNAME: ${{ github.event.commits[0].author.username }}
```

**Comment by Borda** [comment]
This cannot be done the target action audience are forked PRs and GH actions [do not share secrets](https://github.community/t/make-secrets-available-to-builds-of-forks/16166) needed to send a mail.
Some way-around is using public mail server so we do not need to set user/pass as secret or use another tooling 🐰

**Pull request closed without merging**